requirements.txt

```diff
@@ -1,9 +1,10 @@
 numpy>=1.13.3
-pandas>=0.23.0
+pandas>=0.23.0,<0.24.0
```

Comment:
Does this library need to restrict itself to pandas 0.23?

Reply:
not necessary, reverted back to 0.23